driver/config.go

```diff
@@ -28,51 +29,100 @@ type Config struct {
 // NewConfigFromCliContext creates a new config instance from
 // the command line inputs.
 func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
-	jwtSecret, err := jwt.ParseSecretFromFile(c.String(flags.JWTSecret.Name))
-	if err != nil {
-		return nil, fmt.Errorf("invalid JWT secret file: %w", err)
+	// Defaults config
+	cfg := Config{
+		ClientConfig: &rpc.ClientConfig{
+			Timeout: 12 * time.Second,
+		},
+		RetryInterval:         backoff.DefaultMaxInterval,
```

Comment:
We should define the default value in flag definition, and use `flags.BackOffRetryInterval.Value` here